storybook/package.json

```
@@ -10,15 +10,17 @@
 	"scripts": {
 		"prebuild": "npm run remove-dist",
 		"build": "storybook build -c . -o ../dist --quiet --loglevel warn",
-		"start": "storybook dev -p 9001 -c . --no-version-updates",
-		"remove-dist": "rimraf ../dist/"
+		"start": "storybook dev -p 6006 -c . --no-version-updates",
+		"remove-dist": "rimraf ../dist/",
+		"test-storybook": "test-storybook --config-dir ."
```

All scripts in this package are specific to Storybook 🙂 but this is also consistent with changes that were made as part of the recent Node upgrade (I believe – see `test:e2e`)
```suggestion
		"test": "test-storybook --config-dir ."
```